containers/Ideas/IdeaCreate.js

```diff
@@ -161,6 +161,11 @@ export const IdeaCreate = props => {
 
   const onSetImage = (image) => setImage(image);
 
+  const pressHandler = () =>
```

Comment:
it might be good to have a more specific name for this handler (since we are in a container component)